plasma_framework/contracts/src/exits/payment/PaymentExitDataModel.sol

```diff
@@ -26,7 +26,10 @@ library PaymentExitDataModel {
     }
 
     struct InFlightExit {
-        uint256 exitStartTimestamp;
+        uint64 exitStartTimestamp;
+        // Canonicity is assumed at start, then can be challenged and is set to `false`.
+        // Response to non-canonical challenge can set it back to `true`.
+        bool isCanonical;
```

Comment:
Can we add a TODO here? We'd need to set this flag to true while startIFE, since this is new there is nothing in startIFE handling this yet.

Reply:
I did it right away in my `after review` commit